setup.py

```diff
@@ -135,12 +135,12 @@ def get_winver():
             'psutil/arch/windows/inet_ntop.c',
             'psutil/arch/windows/services.c',
             'psutil/arch/windows/global.c',
-            # 'psutil/arch/windows/connections.c',
+            'psutil/arch/windows/wmi.c',
```

Comment:
Question for you: can this be considered WMI related code? I added this new file because some time in the future I am hoping to add sensors/temperatures metrics (which I know require WMI).

Reply:
Yeah it can, performance counters are just an abstraction on top of WMI, check out this architectural diagram here: https://docs.microsoft.com/en-us/windows/desktop/perfctrs/about-performance-counters